packages/types/src/lib/interfaces.ts

```diff
@@ -1055,34 +1081,35 @@ export interface GetSignSessionKeySharesProp {
+  // If you want to pass the `authNeededCallback`, having a PKP public key is necessary. This can be used if you do not have a wallet and wish to authenticate with your PKP.
   pkpPublicKey?: string;
 
-  // When this session signature will expire.  The user will have to reauthenticate after this time using whatever auth method you set up.  This means you will have to call this signSessionKey function again to get a new session signature.  This is a RFC3339 timestamp.  The default is 24 hours from now.
+  // When this session signature will expire. After this time is up, you will need to reauthenticate using the same authentication method, generating a new session signature. The default time until expiration is 24 hours. The formatting is an [RFC3339](https://datatracker.ietf.org/doc/html/rfc3339) timestamp.
   expiration?: any;
 
-  //   The chain to use for the session signature.  This is the chain that will be used to sign the session key.  If you're using EVM then this probably doesn't matter at all.
+  // The chain to use for the session signature and sign the session key. If you're using EVM, this parameter isn't very important.
   chain?: Chain;
 
   /**
    * An array of resource abilities that you want to request for this session. These will be signed with the session key.
-   *
-   * @example If you want to request the ability to decrypt an access control condition, then you would pass
-   * [{ resource: new LitAccessControlConditionResource('someResource), ability: LitAbility.AccessControlConditionDecryption }]
+   * If you want to request the ability to decrypt an access control condition, then you would pass something similar to the example:
+   * @example 
+   * [{ resource: new LitAccessControlConditionResource('someResource`), ability: LitAbility.AccessControlConditionDecryption }]
    */
   resourceAbilityRequests: LitResourceAbilityRequest[];
 
   /**
    * The session capability object that you want to request for this session.
    * If you pass nothing, then this will default to a wildcard for each type of resource you're accessing.
    *
-   * @example If you passed nothing, and you're requesting to perform a decryption operation for an access
+   * If you passed nothing, and you're requesting to perform a decryption operation for an access
    * control condition, then the session capability object will be a wildcard for the access control condition,
    * which grants this session signature the ability to decrypt this access control condition.
    */
   sessionCapabilityObject?: ISessionCapabilityObject;
 
   /**
-   * If you want to ask Metamask to try and switch the user's chain, you may pass true here.  This will only work if the user is using Metamask.  If the user is not using Metamask, then this will be ignored.
+   * If you want to ask MetaMask to try and switch the user's chain, you may pass true here.  This will only work if the user is using MetaMask.  If the user is not using MetaMask, then this will be ignored.
```

Comment:
```suggestion
   * If you want to ask MetaMask to try and switch the user's chain, you may pass true here.  This will only work if the user is using MetaMask, otherwise this will be ignored.
```